prow/cmd/mkbuild-cluster/main.go

```diff
@@ -234,7 +234,7 @@ func describeCluster(o options, kubeContext string) (*describe, error) {
 		if err != nil {
 			return nil, fmt.Errorf("create client: %v", err)
 		}
-		certPEM, keyPEM, caPEM, err := gencert.CreateClusterAuthCredentials(clientset)
+		certPEM, keyPEM, caPEM, err := gencred.CreateClusterCertificateCredentials(clientset)
```

Comment:
Why do we continue to use certs here? We should be able to use an SA token now or not?

Reply:
You are right, we should be able to use SA token now. There will be moderate cleanup in `mkbuild-cluster` to make the switch thus I would prefer to defer  this refactor to a separate PR.

Reply:
@alvaroaleman - I added a task to #13820, to make service account token the default output.